bin/kind-load

```diff
@@ -74,7 +75,7 @@ for img in proxy controller web grafana cli-bin debug cni-plugin ; do
   if [ $images ]; then
     if [ "$images_host" ]; then
       mkdir -p image-archives
-      DOCKER_HOST=$images_host docker save "$DOCKER_REGISTRY/$img:$TAG" > image-archives/$img.tar
+      DOCKER_HOST=$images_host docker save "$DOCKER_REGISTRY/$img" > image-archives/$img.tar
```

Comment:
And this wouldn't change

Reply:
Thanks! This all makes sense and I'm updating the code now